packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -0,0 +1,81 @@
+import {NetworkName, networksChainConfig} from "@lodestar/config/networks";
+import {LCTransport} from "../../../interfaces.js";
+import {CliCommandOptions} from "../../../utils/command.js";
+
+export type StartArgs = {
+  port: number;
+  network: string;
+  "execution-url": string;
+  transport: "rest" | "p2p";
+  "beacon-urls"?: string[];
+  "beacon-bootnodes"?: string[];
+  "ws-checkpoint"?: string;
+};
+
+export type StartOptions = {
```

Comment:
can we change this to {...} & ({ transport:LCTransport.rest,beaconUrls:..} | {transport:LCTransport.p2p,beaconBootnodes:...}